datalad/distribution/tests/test_create_github.py

```diff
@@ -53,10 +53,11 @@
 
 
 def use_cassette(name, *args, **kwargs):
-    """Adapter to store fixtures locally
+    """Adapter to store fixtures locally and skip if there is no vcr
 
-    TODO: RF so could be used in other places as well
+    TODO: RF local aspect so could be used in other places as well
     """
+    kwargs['skip_if_no_vcr'] = kwargs.get('skip_if_no_vcr', True)
```

Comment:
aka `kwargs.setdefault('skip_if_no_vcr', True)`

Reply:
oh hoh hoh -- thanks for the review!  apparently I don't know basics, like features which have been there forever. shame on me.  I will push the fixup.  I bet I used similar pattern in other places as well, heh heh

Reply:
pushed
ha -- only `annotate_path` has similar place to RF... I guess I will leave that one alone to keep this PR pure ;)
```
$> git grep '\<\([^ ]\+\)\[.\(.*\).\] = \1.get(.\2.'
datalad/auto.py:                out['note'] = out.get('note', '').strip()
datalad/interface/annotate_paths.py:                    path_props['parentds'] = path_props.get('parentds', dspath)
datalad/interface/ls_webui.py:                children_size[size_type] = children_size.get(size_type, 0) + machinesize(child_size)
datalad/interface/utils.py:            spec[current_trace[-1]] = spec.get(current_trace[-1], set()).union(
datalad/interface/utils.py:            actsum[res['status']] = actsum.get(res['status'], 0) + 1
```